services/ux-backend/main.go

```diff
@@ -7,12 +7,17 @@ import (
 	"os"
 	"strconv"
 
-	"k8s.io/klog/v2"
-
+	v1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
```

Comment:
pls change to `ocsv1`, we generally use `v1` for native apis, even that I'm not seeing lately.

Reply:
ack